hw/sensor/creator/src/sensor_creator.c

```diff
@@ -93,6 +100,14 @@ static struct sensor_itf i2c_0_itf_lsm = {
 };
 #endif
 
+#if MYNEWT_VAL(I2C_3) && MYNEWT_VAL(MPU6050_OFB)
```

Comment:
For the sensor_creator, we would like to keep everything generic and hence I2C_0 is used, I would keep it in sync with everything and change I2C_3 to I2C_0.